src/graphql/client/index.js

```diff
@@ -7,18 +8,47 @@ import createLink from './link'
 
 let apolloClient = null
 
-const def = (options) => ({
-  getToken,
-  ...options
+// Bump this whenever client-side schema or fields on a query changes
+const SCHEMA_VERSION = '1'
```

Comment:
is there a way to create a schema signature/hash so we don't need to update this manually?

Reply:
We could use an md5 checksum of `src/graphql/resolvers/schema.gql` and `src/graphql/queries`. That'll cover all changes with the trade-off of invalidating caches more often than needed since not all changes break the schema (which is no big deal imo).